deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -1340,11 +1341,8 @@ def check_time_travel(time_travel: bool, interval: timedelta = timedelta(hours=1
         monitor (FileMonitor, optional): if passed, after changing system clock it will check for the end of the
             scheduled scan. The `monitor` will not consume any log line. Default `None`.
         timeout (int, optional): If a monitor is provided, this parameter sets how log to wait for the end of scan.
-
-    Raises
-        TimeoutError: if `monitor` is not `None` and the scan has not ended in the
-            default timeout specified in `global_parameters`.
```

Comment:
I believe this function still raises this error.

Reply:
Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/1ff97d48c736c3eececd80e24a06b8e6de298b9c)